glean-core/src/upload/mod.rs

```diff
@@ -32,6 +32,8 @@ mod policy;
 mod request;
 mod result;
 
+const WAIT_TIME_FOR_PING_PROCESSING: u64 = 500; // in milliseconds
```

Comment:
Python and Swift take the sleep time in seconds, so we convert that, but that results in a 0s sleep. Might still result in enough of a context switch to make this work in the general case, but it seems the 1s we have right now might be better?

Reply:
Ah! I thought it would becaose 0.5s. I'll change it to 1s, makes sense.